common/src/main/scala/io/phdata/services/EmbeddedImageEmail.scala

```diff
@@ -0,0 +1,27 @@
+package io.phdata.services
+
+import courier.Multipart
+import javax.mail.internet.MimeBodyPart
+
+class EmbeddedImageEmail {
+
+  def embedInlineImage(htmlContent: String, imageUrl: String, cid: String): Multipart = {
+    val htmlPart = new MimeBodyPart()
+    htmlPart.setText(htmlContent, "UTF-8", "html")
+
+    val imagePart: MimeBodyPart = new MimeBodyPart()
+    imagePart.attachFile(imageUrl)
+    imagePart.setContentID(s"<$cid>")
+    imagePart.setDisposition("INLINE")
+    val content = Multipart(List(htmlPart, imagePart), "related")
+    content
```

Comment:
You don't have to create a new variable `content`, instead just directly return `Multipart(List(htmlPart, imagePart), "related")`